agenta-cli/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "agenta"
-version = "0.24.2"
+version = "0.24.5a1"
```

Comment:
it seems you forgot to add the new dependencies, no?

Reply:
Ah! yes !